tests/__sf_section_dotnet.test.fish

```diff
@@ -0,0 +1,86 @@
+source $DIRNAME/spacefish_test_setup.fish
+source $DIRNAME/mock.fish
+
+function setup
+	spacefish_test_setup
+	mock dotnet 0 "echo \"2.1.403\""
+	mkdir -p /tmp/tmp-spacefish
+	touch /tmp/tmp-spacefish/project.json
+	cd /tmp/tmp-spacefish
+end
+
+function teardown
+	rm -rf /tmp/tmp-spacefish
+end
+
+test "Prints section"
+	(
+		touch
```

Comment:
This touch is causing a silent error in the tests, just as a heads-up! :slightly_smiling_face: 
![image](https://user-images.githubusercontent.com/26250962/49082095-bd25cd00-f240-11e8-9300-53079045ad7b.png)

Reply:
Should be fixed now